surf_warp/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2020 O.S. Systems Sofware LTDA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "surf_warp"
+version = "0.1.0"
+authors = ["asakiz <asakizin@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+async-trait = "0.1"
+derive_more = { version = "0.99", default-features = false, features = ["from"] }
+futures-util = "0.3"
+rwcst = { path = "..", package = "rust-web-client-server-testing" }
+tokio = { version = "0.2", features = ["macros"] }
+warp = "0.2"
+surf = "2.0.0-alpha.4"
+http-client = "3.0"
+http-types = "2.2"
```

Comment:
Unused?

Please install `cargo-udeps` so you can check it and verify all deps are used.

Reply:
Thanks for the advice

Reply:
`http-types` is reexported by `surf`, It's better to use it than to include the dependency directly to avoid version conflicts.

Reply:
I'll change that.